include/caf/leds.rst

```diff
@@ -10,27 +10,59 @@ CAF: LEDs module
 The LEDs module of the :ref:`lib_caf` (CAF) is responsible for controlling LEDs in response to LED effect set by a ``led_event``.
 The source of such events could be any other module in |NCS|.
 
-The module uses its own variant of Zephyr's :ref:`zephyr:pwm_api` driver for setting the LED color, either RGB or monochromatic.
-For this reason, it requires a manual configuration of the PWM ports and LEDs.
+The module uses Zephyr's :ref:`zephyr:led_api` driver for setting the LED color, either RGB or monochromatic.
+The Zephyr's LED driver can use either GPIO or PWM based implementation.
 
 Configuration
 *************
 
 To use the module, you must enable the following Kconfig options:
 
 * :option:`CONFIG_CAF_LEDS` - This option enables the LEDs module.
-* :option:`CONFIG_PWM` - This option enables Zephyr's :ref:`zephyr:pwm_api` driver, which is used for setting the LED color (that is, the brightness of the diodes).
 * :option:`CONFIG_LED` - This option enables the LED driver API.
-* :option:`CONFIG_LED_PWM` - This option enabled the LED PWM driver.
```

Comment:
Configuration is complex topic right now. I think we should split it into points similarly as in sensor_sampler. Do you agree @greg-fer ?

``` 
To use the module, you must complete the following requirements:

1. Enable the module by following Kconfig options:

* :option:`CONFIG_CAF_LEDS` - This option enables the LEDs module.
* :option:`CONFIG_LED` - This option enables the LED driver API.

#. Chose and configure the driver implementation for controlling LEDs.
   a. PWM - ...
   * :option:`CONFIG_CAF_LEDS_PWM` - ...
   * :option:`CONFIG_LED_PWM` - ....
   * :option:`CONFIG_PWM` - ....
   b. GPIO - ...
   * :option:`CONFIG_CAF_LEDS_GPIO` - ...
   * :option:`CONFIG_LED_GPIO` - ...
   * :option:`CONFIG_GPIO` - ...

```

Reply:
Yes, this is a good point.

Reply:
Changed